tests/BUILD

```diff
@@ -54,9 +54,20 @@ cris_cc_test (
     ],
 )
 
+filegroup(
+    name = "job_runner_test_tsan_suppressions",
+    srcs = ["job_runner_test_tsan_suppressions.txt"],
+)
+
 cris_cc_test (
     name = "job_runner_test",
     srcs = ["job_runner_test.cc"],
+    env = {
+        "TSAN_OPTIONS": "suppressions=$(execpath job_runner_test_tsan_suppressions)",
```

Comment:
Can we apply this at compile time rather than runtime? Would be hard to run it manually.

Reply:
No, that will bring more false positives. See https://clang.llvm.org/docs/ThreadSanitizer.html#ignorelist. The function marks seems to be a better solution (so that manually tests will be easier) but that does not seem to work. Maybe clang-12 does not have this feature for now.